config/app.default.php

```diff
@@ -244,6 +249,8 @@
              * which is the recommended value in production environments
              */
             //'init' => ['SET GLOBAL innodb_stats_on_metadata = 0'],
+            
+            'url' => env('DATABASE_URL', ''),
```

Comment:
FWIW, according to my quick debug, it seems like CakePHP try to parse the empty string.

When I change `app.php` to something like this:
```php
'url' => env('DATABASE_TEST_URL', null),
```

It works as expected.